application/controllers/AuthenticationController.php

```diff
@@ -67,16 +94,22 @@ public function logoutAction()
         if (! $auth->isAuthenticated()) {
             $this->redirectToLogin();
         }
+        $user = $auth->getUser();
         // Get info whether the user is externally authenticated before removing authorization which destroys the
         // session and the user object
-        $isExternalUser = $auth->getUser()->isExternalUser();
+        $isExternalUser = $user->isExternalUser();
         // Call provided AuthenticationHook(s) when logout action is called
-        AuthenticationHook::triggerLogout($auth->getUser());
+        AuthenticationHook::triggerLogout($user);
```

Comment:
The changes you've made here aren't necessary since you don't need `$user` below anymore. Please undo them.